package.json

```diff
@@ -18,61 +18,61 @@
     },
     "private": false,
     "dependencies": {
-        "@angular/platform-server": "^6.1.9",
+        "@angular/cdk": "^7.2.0",
```

Comment:
Why do you add the `cdk`?

Reply:
it's needed to compile `ng build dotcms-js`

Reply:
Are you 100% about this? Because we didn't have this on 6 and were able to compile `dotcms-js`